src/GrumPHP/Task/Phpcs.php

```diff
@@ -84,4 +94,34 @@ public function run(ContextInterface $context)
 
         return TaskResult::createPassed($this, $context);
     }
+
+    /**
+     * @param array $config
+     *
+     * @return null|string
+     */
+    protected function getWhitelistPathPattern(array $config)
+    {
+        /** @var string|null $whitelistPathPattern */
+        $whitelistPathPattern = $config['whitelist_path_pattern'];
+        if (!$whitelistPathPattern) {
+            return null;
+        }
+
+        $whitelistPathPattern = $this->escapePatternDirectorySeparator($whitelistPathPattern);
+        $whitelistPathPattern = '/'.$whitelistPathPattern.'.(%s)$/';
```

Comment:
I think I can not use the Regex util because Symfony component remove the backslashes.    
In the class `\GrumPHP\Configuration\Compiler\TaskCompilerPass::process` at line `$configuration = $container->getParameter('tasks');` for the current configuration    
```yml
whitelist_pattern: "^src\/(.*)"
```
The parsed configuration return 
```php
"whitelist_pattern" => "^src/(.*)"
```
So the regex syntaxe is wrong and I have to manually escape